sensor_msgs/include/sensor_msgs/image_encodings.hpp

```diff
@@ -124,7 +126,6 @@ static inline bool isColor(const std::string & encoding)
          encoding == RGBA8 || encoding == BGRA8 ||
          encoding == RGB16 || encoding == BGR16 ||
          encoding == RGBA16 || encoding == BGRA16 ||
-         encoding == YUV422 || encoding == YUV422_YUY2 ||
```

Comment:
They were replaced by `encoding == UYVY || encoding == YUYV` in the line below. But if we keep the deprecated encodings for one cycle, then this will stay.